pom.xml

```diff
@@ -220,6 +220,16 @@
                 <artifactId>jackson-core</artifactId>
                 <version>${version.jackson}</version>
             </dependency>
+            <dependency>
```

Comment:
I was expecting some kind of (transitive) dependency exclusion. Are you sure the duplicated classes (once from codehaus and once from fasterxml) are gone with this?

Reply:
I'm not seeing anything in `dependency:tree` from codehaus for jackson.  The only issue I saw was that these two jackson components were being directly included by the avro and used a slightly older version.

Reply:
Ah yes. So indeed the issue went away with the CP version update (DBZ-1925). Before that, the two different versions would show up on the Classpath, but CP 5.4.x doesn't pull in the Codehaus version any longer. That said, the issue still is present for the Cassandra connector; in this case Cassandra itself pulls in the older version. For that one I'm not so sure though whether we can simply override this. So I'll log this one separately, and reword DBZ-1943 to match the simple version alignment done here. Thanks!